utils/check-ruleset-path.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This trivial script is created to detect creation of ruleset
+# files outside of the rules directory, i.e. src/chrome/content/rules
+# 
+# A list of known XML
+# 
+#  - chromium/updates-master.xml
+#  - utils/relaxng.xml
+
+
+# Change directory to git root; taken from ../test/test.sh
+if [ -n "$GIT_DIR" ]
+then
+    # $GIT_DIR is set, so we're running as a hook.
+    cd $GIT_DIR
+else
+    # Git command exists? Cool, let's CD to the right place.
+    git rev-parse && cd `git rev-parse --show-toplevel`
+fi
+
+XML_COUNT=`find . -path "./src/chrome/content/rules" -prune -o -name "*.xml" -print | wc -l`
+
+if [ "$XML_COUNT" -ne 2 ]; then
```

Comment:
This seems rather brittle, and (in my working directory) currently innacurate:

```shell
user@https-everywhere ~/workspace/https-everywhere (master) $ find . -path "./src/chrome/content/rules" -prune -o -name "*.xml" -print
./utils/relaxng.xml
./chromium/updates-master.xml
./chromium/updates.xml
```

We should generalize this somehow, either by taking @jeremyn's suggestion in https://github.com/EFForg/https-everywhere/pull/14016#issuecomment-352792000 or by some other method.

It would probably suit us to add some additional labels or re-purpose existing ones as well, and auto-label PRs based on which directories they've placed files in.  It would make detection of problems like we saw in  https://github.com/EFForg/https-everywhere/pull/14010 easier to detect.  For instance: `ruleset`, `utils`, `tests`, `non-ruleset` etc.

Reply:
There's no `chromium/updates.xml` , see https://github.com/EFForg/https-everywhere/blob/master/chromium/updates.xml (404).

Reply:
@jeremyn It is included in https://github.com/EFForg/https-everywhere/blob/master/chromium/.gitignore and generated by https://github.com/EFForg/https-everywhere/blob/fe9f98c88688752cb88bbb328e29e7866683d743/make.sh#L46.  Therefore this test will fail for anyone running it after `./make.sh`